ca.parallel_launcher.ParallelLauncher.json

```diff
@@ -0,0 +1,52 @@
+{
+	"id": "ca.parallel_launcher.ParallelLauncher",
+	"runtime": "org.kde.Platform",
+	"runtime-version": "5.15-21.08",
+	"sdk": "org.kde.Sdk",
+	"command": "parallel-launcher",
+	"modules": [
+		{
+			"name": "parallel-launcher",
+			"buildsystem": "qmake",
+			"config-opts": [ "CONFIG+=flatpak-version" ],
+			"sources": [
+				{
+					"type": "git",
+					"url": "https://gitlab.com/parallel-launcher/parallel-launcher.git",
+					"branch": "a4c436b0c8077fdbb2ee4aeb68b15d95e7bfb5db"
+				},
+				{
+					"type": "file",
+					"path": "ca.parallel_launcher.ParallelLauncher.metainfo.xml"
+				}
+			],
+			"post-install": [
+				"install -D -t /app/share/parallel-launcher/translations/ lang/*.qm",
+				"mkdir -p /app/share/icons/hicolor/128x128/apps/",
+				"mv -f /app/share/icons/appicon.svg /app/share/icons/hicolor/128x128/apps/ca.parallel_launcher.ParallelLauncher.svg",
+				"mv -f /app/share/mime/packages/bps-mime.xml /app/share/mime/packages/ca.parallel_launcher.ParallelLauncher.xml",
+				"install -D -t /app/share/metainfo/ ca.parallel_launcher.ParallelLauncher.metainfo.xml"
+			]
+		},
+		"retroarch/retroarch.json",
+		"retroarch/retroarch-filters-video.json",
+		"retroarch/retroarch-filters-audio.json",
+		"retroarch/retroarch-assets.json",
+		"retroarch/libretro-database.json",
+		"retroarch/libretro-core-info.json"
+	],
+	"finish-args" : [
+		"--socket=x11",
+		"--socket=wayland",
+		"--socket=pulseaudio",
+		"--share=ipc",
+		"--share=network",
+		"--device=all",
+		"--filesystem=host",
+		"--filesystem=home",
```

Comment:
this is really broad filesystem permission, and redundant.

Reply:
All of these permissions are from the RetroArch flatpak, which has already been approved. It requires them because it launches RetroArch

Reply:
after approval a lot of things happen without.

Reply:
This one is needed by RetroArch. It's also needed by Parallel Launcher itself because you can add any directory on your machine as a location to search for roms (optionally recursively).

Reply:
Oh, I see `home` is redundant with `host`. I will fix that.

Reply:
Fixed